paperqa/agents/search.py

```diff
@@ -546,11 +544,11 @@ async def process_file(
                 document=tmp_docs,
             )
 
-            processed += 1
-            if processed == settings.agent.index.batch_size:
+            processed_counter["n_processed_files"] += 1
```

Comment:
Is this the right way to do it?

Reply:
Yeah it's right, but can we call it something like `batched_save_counter` to be more explicit?